### PR TITLE
changed ternary to &&

### PR DESCRIPTION
**Pull request by k3sha7**
removed redundant ternary operator

### CODE REVIEW

**Comment by k3sha7** [comment]
this is my first open source contribution any feedback would be appreciated

**Review by chartgerink — CHANGES_REQUESTED** [review]
Thanks @k3sha7 for the changes! Most of it looks good, and reduces the amount of code a bit.

I'm not 100% sure what motivated you to make these changes. There is no issue for this and the only one I can think of is #546 - where the example relates to somewhat different places.

Please check out my two review comments and revise your PR in the next two weeks if you'd like for this to be considered further. 

Thanks again for taking the time to contribute! 🙏

File: app/core/components/NavbarTabs.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const NavTabs = ({ currentUser, currentWorkspace, session, router, drafts, invit
                     aria-current={tab.current ? "page" : undefined}
                   >
                     {tab.name}
-                    {tab.count ? (
+                    {tab.count && (
```
[the remark this answers is not in the document]
Comment (nasainsbury) [reply app/core/components/NavbarTabs.tsx:74]: I think to fix this you can do `!!tab.count` as opposed to `tab.count`

[the remark this answers is not in the document]
Comment (nasainsbury) [reply app/core/components/NavbarTabs.tsx:74]: See here: https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component

**Comment by chartgerink** [comment]
@k3sha7 friendly reminder that we'll close this issue as stale on Friday if there's no further activity 😊 If you're working on this, let us know!

Also [happy to chat on our Discord](https://discord.gg/SefsGJWWSw) if you're having issues and want to chat about it.

**Comment by chartgerink** [comment]
I am closing this now due to unresponsiveness of the original contributor. We do appreciate the contribution but we can't work through it without your help if there are open questions.

Thanks and hope you're going into a good weekend!

**Comment by k3sha7** [comment]
Hi there,
Thanks for getting back to me, I just stumbled across this issue and since
I knew what to do, I gave it a shot.But after further investigation i found
out that using ternary operator is better in most of the cases. So u can
ignore this PR

**Comment by chartgerink** [comment]
Thanks @k3sha7 - if you'd still like to be credited as a contributor on the repository, feel free to ping me here. We value all contributions, also those that don't end up making it into the code! 🤗

**Pull request closed without merging**